docs/how-to-install.md

```diff
@@ -6,145 +6,170 @@ license: "This software is licensed under the Apache License version 2."
-
-  - If you build Gravitino yourself using the `./gradlew assembleDistribution` command, you can get the compressed Gravitino binary distribution package with the name `gravitino-<version>-bin.tar.gz` in the `distribution` directory with sha256 checksum file `gravitino-<version>-bin.tar.gz.sha256`.
-
-The Gravitino binary distribution package contains the following files:
-
-```text
-|── ...
-└── distribution/package
-    |── bin/
-    |   ├── gravitino.sh                        # Gravitino server Launching scripts.
-    |   └── gravitino-iceberg-rest-server.sh    # Gravitino Iceberg REST server Launching scripts.
-    |── catalogs
-    |   └── hadoop/                             # Apache Hadoop catalog dependencies and configurations.
-    |   └── hive/                               # Apache Hive catalog dependencies and configurations.
-    |   └── jdbc-doris/                         # JDBC doris catalog dependencies and configurations.
-    |   └── jdbc-mysql/                         # JDBC MySQL catalog dependencies and configurations.
-    |   └── jdbc-postgresql/                    # JDBC PostgreSQL catalog dependencies and configurations.
-    |   └── kafka/                              # Apache Kafka PostgreSQL catalog dependencies and configurations.
-    |   └── lakehouse-iceberg/                  # Apache Iceberg catalog dependencies and configurations.
-    |   └── lakehouse-paimon/                   # Apache Paimon catalog dependencies and configurations.
-    |── conf/                                   # All configurations for Gravitino.
-    |   ├── gravitino.conf                      # Gravitino server and Gravitino Iceberg REST server configuration.
-    |   ├── gravitino-iceberg-rest-server.conf  # Gravitino server configuration.
-    |   ├── gravitino-env.sh                    # Environment variables, etc., JAVA_HOME, GRAVITINO_HOME, and more.
-    |   └── log4j2.properties                   # log4j configuration for the Gravitino server and Gravitino Iceberg REST server.
-    |── libs/                                   # Gravitino server dependencies libraries.
-    |── logs/                                   # Gravitino server and Gravitino Iceberg REST server logs. Automatically created after the server starts.
-    |── data/                                   # Default directory for the Gravitino server to store data.
-    |── iceberg-rest-server/                    # Gravitino Iceberg REST server package and dependencies libraries.
-    └── scripts/                                # Extra scripts for Gravitino.
-```
+### Get the Gravitino package
+
+The Gravitino package comprises the Gravitino server and the Gravitino Iceberg REST server.
+You can manage these servers independently or run them concurrently on a single server.
+
+1. Download the latest Gravitino package from [GitHub](https://github.com/apache/gravitino/releases).
```

Comment:
We should direct users to the official ASF distribution page first https://gravitino.apache.org/downloads